{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = [
+  "setuptools >= 40.9.0",
+  "versioneer-518",
+  "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{cookiecutter.repo_name}}"
+version = "0.0.0"
+description = "{{cookiecutter.description}}"
```

Comment:
There's a `keywords` entry too we could get them to populate.

Reply:
added